packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -0,0 +1,515 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from 'material-ui/styles/withStyles';
+import ButtonBase from 'material-ui/ButtonBase';
+import addEventListener from 'material-ui/utils/addEventListener';
+import { fade } from 'material-ui/styles/colorManipulator';
+
+export const style = theme => {
+  const commonTransitionsOptions = {
+    duration: theme.transitions.duration.short,
+    easing: theme.transitions.easing.easeOut,
+  };
+
+  const commonTransitionsProperty = [
+    'width',
+    'height',
+    'box-shadow',
+    'left',
+    'top',
+    'background-color',
+  ];
+
+  const commonTransitions = theme.transitions.create(
+    commonTransitionsProperty,
+    commonTransitionsOptions,
+  );
+
+  const colors = {
+    primary: theme.palette.primary.main,
+    secondary: '#bdbdbd',
+    focused: '#9e9e9e',
+    disabled: '#bdbdbd',
+  };
```

Comment:
These colours need to be taken from the theme so users can modify them - either directly from `theme.palette.grey`, or `colorManipulator`'d version of them.